src/main/kotlin/blackjack/domain/participant/Participants.kt

```diff
@@ -0,0 +1,49 @@
+package blackjack.domain.participant
+
+import blackjack.application.Deck
+import blackjack.domain.participant.state.Name
+import blackjack.domain.participant.state.role.Role
+
+@JvmInline
+value class Participants(private val values: List<Role>) {
+    init {
+        require(values.size >= MINIMUM_NUMBER_OF_PLAYERS) { "참가자는 2명 이상이어야 합니다." }
+        require(values.map { it.name }.toSet().size == values.size) { "참가자의 이름은 중복될 수 없습니다." }
+    }
+
+    constructor(vararg role: Role) : this(role.toList())
+
+    operator fun plus(role: Role): Participants {
+        if (role is Dealer) {
+            val participants = values.toMutableList()
+            participants.add(0, role)
+            return Participants(participants)
+        }
+        return Participants(values.plus(role))
+    }
```

Comment:
Dealer 또한 이미 Role 이기 때문에 타입 캐스팅을 할 필요가 없이 바로 더해도 됩니다 :)
또, + 를 활용해볼 수 있겠네요 😀
```suggestion
    operator fun plus(role: Role): Participants {
        return Participants(values + role)
    }
```

Reply:
네, 조언해 주신대로 아래와 같이 수정해 보았습니다 😊 확인 부탁드립니다 🙏 

- https://github.com/next-step/kotlin-blackjack/commit/38e6616c8cea1030279c5eeaa78a17c56a1b454b